ow_power_system/config/system.cfg

```diff
@@ -28,16 +28,21 @@ voltage_range: 0.1			        # volts
 # values from other planetary rovers, and some are just guesses.
 
 # Baseline values (watts) for constant power draws.
-baseline_power_camera: 0.1
-baseline_power_communications: 0.6
 baseline_power_computing: 0.1
 baseline_power_heating: 1.2
 baseline_power_science_instr: 0.6
 baseline_power_sample_handling: 0.6
 baseline_power_other: 2.2
 
+
 # Power coefficients for dynamically computed power draw.
-wattage_coefficient_lights: 0.3    # power draw (W) per light at max intensity
+power_lights_coefficient: 0.3       # power draw (W) per light at max intensity
+power_camera_controller: 1.0        # power draw of the camera controller
+power_camera_exposed_coefficient: 2.0  # power draw while a camera is exposing
```

Comment:
I suggest `camera_active` .  "exposed" sounds a bit odd to me.

Reply:
How about `power_active_camera`?

Reply:
Yes, as long as the naming is consistent.

Reply:
Done